tests/app/test_telemetry.py

```diff
@@ -174,7 +174,7 @@ def test_failing_executing_command_sends_telemetry_data(
     # The method is called with a TelemetryData type, so we cast it to dict for simpler comparison
     result_command_event = (
         mock_conn.return_value._telemetry.try_add_log_to_batch.call_args_list[  # noqa: SLF001
-            1
+            0
```

Comment:
Why this change in the test was required?
Does it mean that failing command won't send telemetry data?

Reply:
It will, but it will first send error then usage info. The change is caused by the order of callbacks (pre, error, post). By moving usage from pre to post we changed the events order.